dash_labs/plugins/pages.py

```diff
@@ -297,6 +301,53 @@ def _import_layouts_from_pages():
                 )
 
 
+def _register_page_from_markdown_file(page_filename, app):
+    """
+    Extracts and runs dash.register_page() from the "front matter" of a markdown file in the pages folder.
+    Front matter is defined with three dashes:
+    ---
+    dash.register_page(...)
```

Comment:
In Github, this ends up getting rendered as a header. 

I think the true front-matter syntax would have the variables defined with `:`

Instead of executing `dash.register_page`, what if we allowed the `register_page` keyword arguments in the front matter? like this:
```
path: /hello-world
title: Hello World
```
then it's really readable and usable by non-python programmers 🙂

Reply:
and I guess I was thinking that `layout` would be implicit. it would just be the file. but if someone wanted to override it to pass their own arguments in, I suppose they could have:
```
path: /hello-world
title: Hello World
layout=MarkdownAIO(...)

Reply:
Yes, I'll make it so that the layout is implicit, but then it will only apply the  defaults.  I expect it will be more common to specify it like  `layout=MarkdownAIO(...)` so the props can be changed.

We should also have a way to set global defaults for the app.

Reply:
Done